### PR TITLE
Show/Entry status for students

### PR DESCRIPTION
**Pull request by kayladavis**
When the entry period is over:
-  the "submission date" changes to "Not accepting Submissions"
- a label is put on once it's in the judging period
- submisssion button removed

once judging period is over:
Label update to show invited status

Example of shows in different states
image

also solves #184

### CODE REVIEW

File: frontend/src/Student/components/ShowCard.js [hunk 1]
```diff
@@ -18,86 +19,160 @@ const Card = styled.div`
...
+          <h5>
+            {props.show.entries.length}/{props.show.entryCap} Submissions
+          </h5>
         </div>
         <div>
-          Accepting Submissions Until: <Moment format='MMMM Do YYYY'>{props.show.entryEnd}</Moment>
+          {moment().isAfter(moment(props.show.entryEnd)) ? (
+            'Submissions are closed'
+          ) : (
+            <AcceptingSubmissionMessage {...props} />
```
Comment (billdybas) [inline frontend/src/Student/components/ShowCard.js]: I don't think you need to break this out into a separate component. Consider moving the ternary up one level in the DOM tree, and return the parent `<div>` from both "components".

File: frontend/src/Student/components/ShowCard.js [hunk 2]
```diff
@@ -18,86 +19,160 @@ const Card = styled.div`
...
         <div>
-          <h5>{props.show.entries.length}/{props.show.entryCap} Submissions</h5>
+          <h5>
+            {props.show.entries.length}/{props.show.entryCap} Submissions
+          </h5>
         </div>
         <div>
-          Accepting Submissions Until: <Moment format='MMMM Do YYYY'>{props.show.entryEnd}</Moment>
+          {moment().isAfter(moment(props.show.entryEnd)) ? (
+            'Submissions are closed'
```
Comment (billdybas) [inline frontend/src/Student/components/ShowCard.js]: "Submissions Closed" or "No Longer Accepting Submissions"

File: frontend/src/Student/components/ShowCard.js [hunk 3]
```diff
@@ -18,86 +19,160 @@ const Card = styled.div`
...
-            md='3'
-            className='text-center align-self-center d-flex justify-content-center align-items-center'
-            style={{minHeight: '10em'}}
-            title={entry.title}
-            key={entry.id}>
-            <EntryThumb entry={entry} />
-          </Col>
-        ))
-      }
+      {moment().isAfter(moment(props.show.entryEnd)) ? (
```
Comment (billdybas) [inline frontend/src/Student/components/ShowCard.js]: How about you always render `<SubmittedEntries>` and conditionally add `<NewSubmission>`

File: frontend/src/Student/components/ShowCard.js [hunk 4]
```diff
@@ -18,86 +19,160 @@ const Card = styled.div`
...
+const EntryContainer = styled.div``
+const JudgingPhase = styled.div`
+  position: relative;
+  border: 1px solid transparent;
+  border-radius: 0.25rem;
+  color: #856404;
+  background-color: #fff3cd;
+  border-color: #ffeeba;
+`
+const Accepted = styled.div`
```
Comment (billdybas) [inline frontend/src/Student/components/ShowCard.js]: I would make all of these labels the same width – in the image you attached, make them all the size of `JudgingPhase`, or make them all be 100% width of their container (or in the case of photos, same width as the photo – which I think is 100% of the container)

Comment (kayladavis) [reply frontend/src/Student/components/ShowCard.js]: Width is now the width of the col

image

**Comment by billdybas** [comment]
I'd rebase off of master before pushing.

**Pull request merged**